core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -3116,6 +3116,14 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
 //          OperandTypes.family(SqlTypeFamily.ANY, SqlTypeFamily.STRING),
 //          SqlFunctionCategory.STRING);
 
+  @LibraryOperator(libraries = {TERADATA})
+  public static final SqlFunction JSON_CHECK =
+      new SqlFunction("JSON_CHECK",
+          SqlKind.OTHER_FUNCTION,
+          ReturnTypes.VARCHAR_2000_NULLABLE, null,
```

Comment:
@shirin-hasan i have removed NULLABLE.Because JSON_CHECK doesn't support NULL as operand and i have added all these changes into another MR.Please have a look.
https://github.com/datametica/calcite/pull/675/files

Reply:
It seems that the changes for JSON_CHECK are also incorporated in the MR https://github.com/datametica/calcite/pull/675/files
Which one are we planning to merge?